Mathlib/AlgebraicTopology/SimplicialSet.lean

```diff
@@ -84,6 +84,63 @@ scoped[Simplicial] notation3 "Δ[" n "]" => SSet.standardSimplex.obj (SimplexCat
 instance : Inhabited SSet :=
   ⟨Δ[0]⟩
 
+section Product
+
+/-- `Prod X Y` is the explicit binary product of simplicial sets `X` and `Y`. -/
+def Prod (X Y : SSet) : SSet where
+  obj n := X.obj n × Y.obj n
+  map f a := (X.map f a.1, Y.map f a.2)
+
+/-- The first projection of `Prod X Y`, onto X. -/
+@[simp]
```

Comment:
I don't think this should be tagged with `simp`. Did you mean `simps`? (Similarly for `Prod.snd`)